NetCoreRestApi/DataLayer/Converters/ProductEntityConverter.cs

```diff
@@ -0,0 +1,68 @@
+﻿using DataLayer.Models;
+using Common.Converter;
+using DataLayer.Entities;
+using System.Collections.Generic;
+
+namespace DataLayer.Converters
+{
+    public class ProductEntityConverter : IConverter<ProductModel, ProductEntity>
```

Comment:
Rename to ProductModelConverter
Swap in the interface

Reply:
done